week2/2-thursday/homework/exercise6/RobPEx6.js

```diff
@@ -0,0 +1,24 @@
+var people = [
+    'Dom',
+    'Lyn',
+    'Kirk',
+    'Autumn',
+    'Trista',
+    'Jesslyn',
+    'Kevin',
+    'John',
+    'Eli',
+    'Juan',
+    'Robert',
+    'Keyur',
+    'Jason',
+    'Che',
+    'Ben'
+  ];
+
+  var goodie = people.map(function(greet1){
+      return 'Good Job, ' + greet1 + '!';
+
+  });
```

Comment:
I think a forEach might with a console.log might be a better choice but this work.